apps/ewallet_db/lib/ewallet_db/token.ex

```diff
@@ -160,6 +174,7 @@ defmodule EWalletDB.Token do
         :symbol_first,
         :html_entity,
         :iso_numeric,
+        :locked,
```

Comment:
A mintable token can be locked at some point to prevent any further minting (like the OMG token). So `locked` can be updated from `false` to `true` but never from `true` to `false`.

Reply:
Thinking we might want to change the `locked` to a timestamp. `nil` at creation, and set the timestamp once locked, then immutable. Thoughts?